src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    private String name = "Bulldozer";
```

Comment:
in this case it's better to make it a constant. I mean add static and final keywords